shell/gpu/gpu_surface_metal_impeller.mm

```diff
@@ -125,7 +130,7 @@
         }
 
         if (!disable_partial_repaint && damage) {
-          uintptr_t texture = reinterpret_cast<uintptr_t>(last_texture);
+          uintptr_t texture = reinterpret_cast<uintptr_t>(strongLastTexture);
```

Comment:
Is this safe without a bridge-cast to `void *` inside the reinterpret cast?

Reply:
Nope. Done (everywhere).